### ISSUE TITLE
There is no argument given that corresponds to the required formal parameter 'bundles' 

### ISSUE THREAD

**Issue by Zero2Cool** [YAFNET/YAF.SampleWebApplication#33]
Downloaded latest source for Sample Application and getting an error about bundles. 

Error	CS7036	There is no argument given that corresponds to the required formal parameter 'bundles' of 'BundleConfig.RegisterBundles(BundleCollection)'	YAF.SampleWebApplication	

I believe the fix is to replace the current Bundle line with the following.
`BundleConfig.RegisterBundles(BundleTable.Bundles);`

![image](https://user-images.githubusercontent.com/5612416/143046043-f8d66d20-597a-409c-8255-b46047ab28c6.png)

Update: Yes, that line gets the solution to compile, however, I encountered another error regarding `'jquery' is not a valid script name. The name must end in '.js'` and I'm just not able to keep trying to fix this. I assume you ran this solution from the source and have no issues and I just don't have the time to keep playing whack-a-mole trying to fix all these issues. Sorry.

**Comment by w8tcha** [comment]
Sorry looks like i forgot to update 2 files in the app_start folder.

**Comment by Zero2Cool** [comment]
Copied those two files and now able to view the forums, but things like drop down menus do not work.  And seeing this in Chrome console, if it helps.
![image](https://user-images.githubusercontent.com/5612416/143253924-4abcf06a-f547-426a-8bec-7f564bdb18d6.png)

**Comment by Zero2Cool** [comment]
First, I got YAF to install with clean database.
Then I got YAF to upgrade my existing database while being able to login without resetting passwords.
Then I got YAF.SampleApplication to install with clean database.
Then I got YAF.SampleApplication  to upgrade my existing database while being able to login without resetting passwords.

And now it looks like my four day weekend is going to be modifying YAF source to a layout I prefer and upgrade to v3.0.1. Cheers!
